compiler/src/dotty/tools/dotc/decompiler/DecompilationPrinter.scala

```diff
@@ -40,12 +40,8 @@ class DecompilationPrinter extends Phase {
 
   private def printToOutput(out: PrintStream)(using Context): Unit = {
     val unit = ctx.compilationUnit
-    if (ctx.settings.printTasty.value)
```

Comment:
Should probably delete or at least deprecate the setting, no?  I think it's unused with these changes.

Reply:
The setting still works. It is just handled before it gets to the decompilation printer. Users still need to write `scalac -print-tasty`.

Reply:
It looked to me like it's matched and dropped (shift) in `scalac` - but I didn't run it to be sure.

Reply:
Yes, but now it calls the `$TastyPrinterMain` directly instead of going through `$DecompilerMain`. That is why we do not have to handle it in the `DecompilationPrinter` anymore.

Reply:
Yep.  I still think the setting is now redundant, but maybe we'll end up re-setting it if we switch TastyPrinter back to using `CliCommand.distill`.

Reply:
One reason to keep it in the settings is to have it listed when a user does `scalac -help`.